### PR TITLE
v1.5.0

### PR DESCRIPTION
**Pull request by etaletai13**
screen shot 2018-04-30 at 10 37 40 am

### CODE REVIEW

**Comment by etaletai13** [comment]
I'd recommend just testing out the repo I've got and then we can figure out the conflicts after. It's mostly removals.

**Comment by Varunram** [comment]
Will test and merge later today.

File: package.json [hunk 1]
```diff
@@ -27,15 +44,17 @@
...
-    "snyk-protect": "snyk protect",
-    "prepare": "npm run snyk-protect",
-    "test": "snyk test"
+  "devDependencies": {
+    "nwjs-builder-phoenix": "^1.14.6"
   },
-  "snyk": true
+  "scripts": {
+    "build:app": "build --concurrent --tasks win-x86,winx64,linux-x86,linux-x64,mac-x64 --mirror https://dl.nwjs.io/ .",
+    "postinstall": "mkdir -p bin/gmc && cp node_modules/\\@musicoin/gmc/bin/gmc bin/gmc/ && mv node_modules/\\@musicoin/ux ./interface && yarn secondstage",
```
Comment (Varunram) [inline package.json]: I get this error log while trying to just `yarn`, so I'm assuming that this package wasn't included in the yarn dependencies or something similar.

```
cp: node_modules/@musicoin/gmc/bin/gmc: No such file or directory
error Command failed with exit code 1.
```

Comment (Varunram) [reply package.json]: Also, checkout the child repo for an issue I created just now.

**Comment by Varunram** [comment]
@etaletai13 Apologies for the late review, was caught up in a few things and count find time to review your PR. The PR looks great btw, adding modularity and stuff is something that we like to do and I think this would be great, going into v1.5.0. Could you post your email address here or send an email from your account to `varunram@musicoin.org`? I'd like to invite you to our slack channel so that you can be more closely involved with the future iterations of the wallet and we can move things forward from there.

File: BOUNTY.md [hunk 2]
```diff
@@ -1,24 +1,21 @@
 # The Musicoin Bounty Program
```
[the remark this answers is not in the document]
Comment (etaletai13) [reply BOUNTY.md]: I definitely need to get my thinking a little more linear, to be more clear for everyone. I tend to do something, and then think better about it later on. Which wouldn't be too hard to clean up. Thanks!

File: README.md [hunk 3]
```diff
@@ -1,42 +1,26 @@
...
+[![GitHub Release Date](https://img.shields.io/github/release-date/Musicoin/desktop.svg?style=for-the-badge)](https://github.com/Musicoin/desktop/releases) 
+[![Github All Releases](https://img.shields.io/github/downloads/Musicoin/desktop/total.svg?style=for-the-badge)](https://github.com/Musicoin/desktop/releases) 
+![GitHub contributors](https://img.shields.io/github/contributors/Musicoin/desktop.svg?style=for-the-badge) 
 
-The official Musicoin Desktop Wallet
-
-## Nightly Builds
+---
 
-Nightly builds for all supported Operating Systems can be found over at [builder.musicoin.org](https://builder.musicoin.org/)
```
[the remark this answers is not in the document]
Comment (etaletai13) [reply README.md]: Good call, no actual reason for that.

**Comment by etaletai13** [comment]
closed In favor of: #245

**Pull request closed without merging**